libs/vault/src/cipher-form/components/item-details/item-details-section.component.ts

```diff
@@ -219,14 +231,47 @@ export class ItemDetailsSectionComponent implements OnInit {
     if (!this.allowOwnershipChange) {
       this.itemDetailsForm.controls.organizationId.disable();
     }
+    await this.setFormStatus(true);
     this.itemDetailsForm.controls.organizationId.valueChanges
       .pipe(
         takeUntilDestroyed(this.destroyRef),
-        concatMap(async () => await this.updateCollectionOptions()),
+        concatMap(async () => {
+          await this.updateCollectionOptions();
+          // Prevent infinite loop if the form is already enabled as enabling the form causes valueChanges to emit
+          if (this.formEnabled) {
+            return;
+          }
+          await this.setFormStatus();
+        }),
       )
       .subscribe();
   }
 
+  private async setFormStatus(init = false) {
```

Comment:
🎨 (non-blocking) - IMO a comment would be helpful here to understand the use case without having to find the JIRA ticket. 


Maybe something like this?
```suggestion
  /**
   * When there is the cipher does not belong to an organization but the user is a part of
   * an organization that requires all ciphers to be owned by an organization, disable the entire form
   * until the user selects an organization.
   */
  private async setFormStatus(init = false) {
```